apps/android/settings.js

```diff
@@ -1,4 +1,13 @@
 (function(back) {
+
+  function onGpsOverwriteChange(newValue) {
+    if (newValue) {
+      Bangle.setGPSPower(false, 'android');
```

Comment:
I'm not sure this is needed... The way `setGPSPower` works this would only ever turn off the GPS if it had been turned on with `Bangle.setGPSPower(true, 'android');` - if any other app requested it, it'd stay turned on